test/Microsoft.ML.Tests/OnnxConversionTest.cs

```diff
@@ -309,6 +309,40 @@ public void BinaryClassificationTrainersOnnxConversionTest()
             Done();
         }
 
+        [Fact]
```

Comment:
This shouldn't be part of your changes. Maybe you did your merge steps wrong?

Reply:
Yeah, I think I messed up, but it's fixed now